multi-node-k8s.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+set -e -o pipefail
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
```

Comment:
isnt it better to specify a version here ?

Reply:
Fetching a version will keep the script up to date with the latest stable version. But yes, we can also update it manually and skip this.